packages/autosuggest-react/package.json

```diff
@@ -0,0 +1,61 @@
+{
+    "name": "@fremtind/jkl-autosuggest-react",
```

Comment:
Jeg lurer på om dette trenger å være en egen pakke. Hvorfor ikke ha det som en del av TextInput?

Reply:
Tenker egentlig også det, men som en egen komponent i så fall. Blir mange pakker etter hvert 😅

Et argument for å beholde det som det er, er å kunne gjøre breaking changes (som vi kommer til å måtte) i denne pakken uten å måtte tvinge en major update på de som bruker den "vanlige" komponenten.

Reply:
Er mange pakker et problem? Jeg syns det er mer forvirrende at enkelte komponenter plutsetlig ligger under en annen.

Tenker også det er godt argument å isolere for breaking changes. Men ja, om det er konsensus for å flytte, så flytter jeg.

Reply:
Flyttet den inn i textinput.